pkg/mover/chart.go

```diff
@@ -255,13 +256,55 @@ func (cm *ChartMover) loadChartFromIntermediateBundle(bundlePath string) error {
 
 // loadChartFromPath load the chart in memory from a given path
 func (cm *ChartMover) loadChartFromPath(path string) error {
-	var err error
-	if cm.chart, err = loader.Load(path); err != nil {
-		return &ChartLoadingError{Path: path, Inner: err}
+	// always unpack the chart tarball to a directory to avoid issues loading
+	// files if the tgz contains replicated entries
```

Comment:
could you add a link to the issue with duplicates?

Reply:
you mean a JIRA link here?

Reply:
ah ok!

adding...

Reply:
done now